tests/Type/data/request-object.php

```diff
@@ -14,5 +14,3 @@
 assertType('Illuminate\Routing\Route|null', $request->route());
 assertType('object|string|null', $request->route('foo'));
 assertType('object|string|null', $request->route('foo', 'bar'));
-
-assertType('App\User|null', $request->user());
```

Comment:
Why was this removed?

Reply:
I moved it to be with the other `$request->user` tests:

![image](https://github.com/larastan/larastan/assets/4176520/d61ef58e-26ac-46e7-b16f-bb4db6a7a263)